app/controllers/mixins/more_show_actions.rb

```diff
@@ -27,6 +27,12 @@ def show_compliance_history
       @showtype = @display
     end
 
+    def show_topology
+      @showtype = "topology"
+      drop_breadcrumb(:name => @record.name + _(" (Topology)"),
+                      :url  => "/#{controller_name}/show/#{@record.id}?display=topology")
+    end
```

Comment:
we have already `show_topology` in EmsCommon. But never mind this can be unified in a follow up PR. But please, use the `show_link` to generate the show links so that this works also for `restful? == true`.